rusk-abi/src/lib.rs

```diff
@@ -27,6 +28,21 @@ pub const POSEIDON_TREE_DEPTH: usize = 17;
 /// and verifier.
 pub const TRANSCRIPT_LABEL: &[u8] = b"dusk-network";
 
+/// The smallest unit of payable Dusk.
+pub type Lux = u64;
```

Comment:
This is not exactly as discussed. I would avoid to use `Lux` at the moment. Here we'll trying to do it properly, with actual type, and then it make sense, or we just stick to `Dusk`. Basically my idea was to do not expose the internal, in this case `Lux`, as much as possible. Otherwise it doesn't even make sense using the `dusk` const fn. 

I mean, if I have something like: `dusk(1_000)` I'm implicitly thinking that what I get back is `dusk`, no matter the unit, where if I have an "additional type" it's strange that a "dusk" function returns a "lux" type: in this case, the aliasing is counter productive.

I would stick to only `Dusk` unit until we see how to do it properly with different types.